src/core/Table/Table.test.tsx

```diff
@@ -1486,3 +1487,51 @@ it('should render sub-rows with custom expander', () => {
   rows = container.querySelectorAll('.iui-table-body .iui-row');
   expect(rows.length).toBe(10);
 });
+
+it('should edit cell data', () => {
+  const onCellEdit = jest.fn();
+  const columns: Column<TestDataType>[] = [
+    {
+      Header: 'Header name',
+      columns: [
+        {
+          id: 'name',
+          Header: 'Name',
+          accessor: 'name',
+          cellRenderer: (props) => (
+            <EditableCell {...props} onCellEdit={onCellEdit} />
+          ),
+        },
+        {
+          id: 'description',
+          Header: 'description',
+          accessor: 'description',
+        },
+        {
+          id: 'view',
+          Header: 'view',
+          Cell: () => {
+            return <span>View</span>;
+          },
+        },
+      ],
+    },
+  ];
+  const { container } = renderComponent({
+    columns,
+  });
+
+  const rows = container.querySelectorAll('.iui-table-body .iui-row');
+  assertRowsData(rows);
+
+  const editableCells = container.querySelectorAll(
+    '.iui-cell[contenteditable]',
+  );
+  expect(editableCells).toHaveLength(3);
+
+  fireEvent.input(editableCells[1], {
+    target: { innerText: 'test-data\n\r\r\n' },
```

Comment:
why new line is needed in inner text?

Reply:
![image](https://user-images.githubusercontent.com/36186912/132663205-f77958ee-aaea-4e82-a2c3-7956e3a2abdf.png)

Reply:
oh...
Could test both cases then, but minor, I guess..

Reply:
Added another test with edge cases.